CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+#Package information
+hyro_init_package(signal_generator VERSION 0.0.1)
+
+find_package ( Qt4 COMPONENTS QtCore QtGui QtOpenGl )
```

Comment:
Do you use Qt in your project? If not, remove Qt related code.

Reply:
@joaobrittoneto I needed to use this for the plot2d. When I comment this, I can't plot.

Reply:
Probably, it is missing something. We need to check, because in theory, **hyro_widgets** includes QT dependencies. Moreover, for plot2d we don't **QtOpenGL**.